apis/apps/pub/inplace_update.go

```diff
@@ -61,6 +61,23 @@ type InPlaceUpdateState struct {
 
 	// UpdateEnvFromMetadata indicates there are envs from annotations/labels that should be in-place update.
 	UpdateEnvFromMetadata bool `json:"updateEnvFromMetadata,omitempty"`
+
+	// NextContainerImages is the containers with lower priority that waiting for in-place update in next batch.
+	NextContainerImages map[string]string `json:"nextContainerImages,omitempty"`
+
+	// NextContainerImages is the containers with lower priority that waiting for in-place update in next batch.
+	NextContainerRefMetadata map[string]metav1.ObjectMeta `json:"nextContainerRefMetadata,omitempty"`
```

Comment:
plz correct the comment about NextContainerRefMetadata

Reply:
updated